.github/workflows/build.yml

```diff
@@ -2,8 +2,6 @@ name: Build and upload binaries
 on:
   release:
     types: [published]
-  push:
```

Comment:
Removing the `push` trigger is more than this commit tries to justify in the commit message & PR details.